tests/unit/test_class_cmds.py

```diff
@@ -525,6 +523,21 @@ class TST_MemberOfFamilyCollection {
                  '<METHOD NAME="DeleteNothing" PROPAGATED="false"'],
       'test': 'regex'},
      SIMPLE_MOCK_FILE, OK],
+
+    ['Verify class command enumerate with --association filter.',
+     ['enumerate', '--association', '--no'],
```

Comment:
Use long option names for clarity: `--no` -> `--names-only`

Reply:
DONE